Make sure test_amber_clean.sh dosen't try to cd into a possibly missing directory.
--- test/test_amber_clean.sh.orig	2025-07-14 13:34:48.866262000 +0200
+++ test/test_amber_clean.sh	2025-07-14 13:35:44.941307000 +0200
@@ -1273,7 +1273,11 @@
 echo $delete_list | xargs /bin/rm -f
 
 # Remove cuda remd test output
-cd cuda/remd && make clean && cd ../..
+if [ -d "cuda/remd" ]; then
+    cd cuda/remd && make clean && cd ../..
+fi
 
 # Remove hip remd test output
-cd hip/remd && make clean && cd ../..
+if [ -d "hip/remd" ]; then
+    cd hip/remd && make clean && cd ../..
+fi
